subsys/bluetooth/services/CMakeLists.txt

```diff
@@ -28,4 +28,5 @@ zephyr_sources_ifdef(CONFIG_BT_DDFS ddfs.c)
 zephyr_sources_ifdef(CONFIG_BT_MDS mds.c)
 add_subdirectory_ifdef(CONFIG_BT_CGMS cgms)
 add_subdirectory_ifdef(CONFIG_BT_FAST_PAIR fast_pair)
+add_subdirectory(ras)
```

Comment:
should this be `add_subdirectory_ifdef(CONFIG_BT_CHANNEL_SOUNDING ras)` or similar?

Reply:
or maybe

add_subdirectory_ifdef(CONFIG_BT_RAS_RREQ ras)
add_subdirectory_ifdef(CONFIG_BT_RAS_RRSP ras)

Reply:
Good point, that should be a dependency of the Kconfigs too